portal-frontend/src/app/features/public/notification/submission/additional-information/additional-information.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { PublicNoticeOfIntentSubmissionDto } from '../../../../../services/public/public-notice-of-intent.dto';
+import { PublicDocumentDto } from '../../../../../services/public/public.dto';
+import { PublicService } from '../../../../../services/public/public.service';
+import { DOCUMENT_TYPE } from '../../../../../shared/dto/document.dto';
+import {
+  RESIDENTIAL_STRUCTURE_TYPES,
+  STRUCTURE_TYPES,
+} from '../../../../notice-of-intents/edit-submission/additional-information/additional-information.component';
+
+@Component({
+  selector: 'app-additional-information',
+  templateUrl: './additional-information.component.html',
+  styleUrls: ['./additional-information.component.scss'],
+})
+export class AdditionalInformationComponent implements OnInit {
+  firstQuestion = 'FIX THIS';
```

Comment:
I remember seeing this on some other feature. Is it intentional here or copy-paste?

Reply:
This is intentional, its a scary placeholder that you will notice if you don't set it properly.